pom.xml

```diff
@@ -217,6 +217,23 @@
             <artifactId>debezium-revapi</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-connector-mysql</artifactId>
+            <version>${version.debezium}</version>
+        </dependency>
+        <dependency>
```

Comment:
IMHO this dependency is not needed as it is a transitive one for `debezium-connector-mysql`